data-propagation.md

```diff
@@ -0,0 +1,25 @@
+# Filecoin Data Propogation
```

Comment:
@whyrusleeping WHYYYYYYY :b

Reply:
PRs welcome. (somehow this file missed out on my global find and replace...)

Reply:
https://github.com/filecoin-project/specs/pull/161 :D <3